lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -343,6 +344,9 @@ main h6 {
 /* 11. Media Queries */
 @media (min-width: 1024px) {
   /* Styles for larger screens */
+  #navigation-toggle {
+    display: none;
```

Comment:
Thanks for the PR! I love the pointer improvement, but I think the navigation toggle design needs a more thorough rework. So can we drop this part for now?

Reply:
Do you want me to take a stab at this thorough rework of the navigation bar/toggle?

Reply:
If you can help, I'd definitely appreciate it 🙏 

IMO, on narrower screens, a sticky sidebar should appear to hold the toggle button and maybe a few other info (e.g. logo, search bar, page title...etc.). For the first iteration tho, just having the toggle button would be good already. We can add other stuff later.

### Examples

#### Rust doc (width <= 700px)

<img width="40%" alt="Screenshot 2024-09-08 at 13 09 35" src="https://github.com/user-attachments/assets/90de2238-b182-4acc-83d4-d23bd0b69190">

#### Python doc (width < 1024px)

<img width="40%" alt="Screenshot 2024-09-08 at 13 13 10" src="https://github.com/user-attachments/assets/d33e8de1-9a75-4c02-879c-2dd9e81bb066">